acceptance/testdata/builder/container/cnb/buildpacks/hello_world/0.0.1/bin/detect

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
We could remove this file I think, since we're not testing detect here :)

Reply:
Deleted.